drivers/watchdog/wdt_nrfx.c

```diff
@@ -71,29 +42,24 @@ static int wdt_nrf_setup(struct device *dev, u8_t options)
 	}
```

Comment:
Is it actually needed? The specification says that the watchdog will force the internal LFRC oscillator when no other LF clock source is active.

Reply:
You are right